patches/minecraft/net/minecraft/core/Direction.java.patch

```diff
@@ -0,0 +1,14 @@
+--- a/net/minecraft/core/Direction.java
++++ b/net/minecraft/core/Direction.java
+@@ -47,6 +_,11 @@
+     private final Direction.AxisDirection axisDirection;
+     private final Vec3i normal;
+     private static final Direction[] VALUES = values();
++    // In vanilla updates are notified only on the horizontal plane, in Forge we also notify up and down
++    private static final Direction[] UPDATE_ORDER = Stream.concat(Plane.HORIZONTAL.stream(), Plane.VERTICAL.stream()).toArray(Direction[]::new);
++    public static final Direction[] getUpdateOrder() {
++        return UPDATE_ORDER.clone();
```

Comment:
Oh wow, so this exposes a stupid decision made ages ago.
This needs to be a list or iterable not an array. Because we don't want to keep cloneing the array every time it's called.
So keep the change from future versions where it's a List

Reply:
I made it an array because I couldn't use `?:` in the Level patch otherwise. How should I deal with that?

Reply:
Or should I backport the values view? Just thought of that.